Emu68Info/AsmFuncs.a

```diff
@@ -12,7 +12,7 @@ _LVOSupervisor EQU -30
 
 ;----------------------------------------------------------
 
-	SECTION __merged,code
+	SECTION .text
```

Comment:
Is this change going to be a problem? It's required in order to get GCC to link the VASM compiled code

Reply:
No problem, tested with SAS/C : works ok.